generators/symfony/templates/circleci.yaml.ejs

```diff
@@ -67,13 +67,15 @@ jobs:
       - run:
           name: PHP-CS-Fixer
           command: vendor/bin/php-cs-fixer fix --dry-run --diff --using-cache no
+          environment:
+            PHP_CS_FIXER_IGNORE_ENV: '1'
```

Comment:
I'm not entirely satisfied having to disable php-cs-fixer version check like that but I don't want to wait for their support of PHP8.2 either.